File: Part1Js/MAX_NUM.js
const MAX_NUM = (arr) => {
  let max = arr[0];
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] > max) {
      max = arr[i];
    }
  }
  console.log(max);
};
console.log(MAX_NUM([1, 2, 36, 5, 4, 3, 2]));

File: Part1/Max_Num.java
package Part1;

class Max_Num {
    public static void main(String[] args) {
        int arr[] ={12,3,4,5,4,3,3,23,32,3};
        int max=arr[0];
        for(int i=0;i<arr.length;i++){
            if(arr[i]>max){
                max=arr[i];
            }
        }
        System.out.println(max);
    }
}